source/_integrations/aws.markdown

```diff
@@ -38,7 +38,7 @@ aws:
       aws_secret_access_key: AWS_SECRET
   notify:
     # use the first credential defined in aws integration by default
-    - service: lambda
+    - action: lambda
```

Comment:
```suggestion
    - service: lambda
```
This has not changed in the `aws` integration schema.

Reply:
Nice catch!